### PR TITLE
Feature/add cards component

### LINKED ISSUE #30: Create cards component
- [x] style card container
- [x] style cards
- [ ] Create card articles

### PR DESCRIPTION
**Pull request by Atos20**
#### What’s this PR do?  
 - It creates a card container and styles the card element 

#### Where should the reviewer start?  
 `Cards.js` and `Cards.scss`
#### How should this be manually tested?  
 run `npm start`
#### Any background context you want to provide?  
 n/a
#### What are the relevant tickets?  
 closes #30 
#### Screenshots (if appropriate) 
 Screen Shot 2020-10-07 at 3 13 55 PM

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -13,7 +12,6 @@ class App extends Component{
     return (
      <>
       <Homepage />
-      <WelcomingSection />
```
Comment (Atos20) [inline src/App/App.js:16]: by removing the welcomingSection and inserting it into the home page it allows us to have more control when we get to routing

File: src/Homepage/Homepage.js [hunk 2]
```diff
@@ -24,6 +27,9 @@ export class Homepage extends Component {
           </div>
           <button className="Login-button">Login</button>
         </nav>
+        <WelcomingSection />
+        <Cards />
+        </>
```
Comment (holladayian) [inline src/Homepage/Homepage.js:32]: Nice use of the React.Fragment `</>` short syntax! Love this

File: public/index.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <link href="https://fonts.googleapis.com/css2?family=Condiment&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500&display=swap" rel="stylesheet">
-
+    <link href="https://fonts.googleapis.com/css2?family=Prompt:wght@100;200;300;400;500&display=swap" rel="stylesheet
```
Comment (holladayian) [inline public/index.html:8]: Did the end of this get cut off? I can't quite tell. But hey, if it works, it works!

**Comment by holladayian** [comment]
Looks pretty good! Let's not forget to make scss mixins and variables if we are going to use scss files!

**Pull request merged**